core/src/saros/net/ResourceFeature.java

```diff
@@ -0,0 +1,34 @@
+package saros.net;
+
+import java.util.Optional;
+
+/** Identifier of Saros Resource Features */
```

Comment:
Was the old reference to the XMPP specification (`<p>See also: XEP-0030: Service Discovery (http://xmpp.org/extensions/xep-0030.html)`) useful?

If so (and if this is the right position for it), you could consider adding it here as well as the constants still server the same purpose.

Reply:
This is a remark which is more related to the what the DiscoveryService is doing, but I don't think it's important to explicitly cite it.